registry/auth/basic/access.go

```diff
@@ -0,0 +1,90 @@
+// Package basic provides a simple authentication scheme that checks for the
+// user credential hash in an htpasswd formatted file in a configuration-determined
+// location.
+//
+// This authentication method MUST be used under TLS, as simple token-replay attack is possible.
+package basic
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	ctxu "github.com/docker/distribution/context"
+	"github.com/docker/distribution/registry/auth"
+	"golang.org/x/net/context"
```

Comment:
Why is this version of context used here? In other parts of the code only distribution/context is used.